conda_build/utils.py

```diff
@@ -101,6 +100,10 @@ def glob(pathname, recursive=True):
     from scandir import walk
 
 
+class BuildLockError(Exception):
```

Comment:
This probably belongs in the exceptions.py file, but I'm not too picky.

Reply:
Sure, let me move it.